sos/plugins/manila.py

```diff
@@ -14,31 +14,28 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
+import os
 from sos.plugins import Plugin, RedHatPlugin
 
 
 class ManilaPlugin(Plugin, RedHatPlugin):
     """Openstack Manila"""
     plugin_name = "manila"
     profiles = ('openstack', 'openstack_controller', 'openstack_manila')
-    option_list = [("cmds", "gathers openstack manila commands", "slow", False)]
+    option_list = [("nas", "List NAS shares", "fast", False)]
 
     def setup(self):
-        if self.get_option("cmds"):
+        if self.get_option("nas"):
```

Comment:
Why not just call this `list`? Since that is obviously what `manila` users are going to expect...